src/scss/variables.scss

```diff
@@ -0,0 +1,347 @@
+$primary: #175DDC;
+$primary-accent: #1252A3;
+$secondary: #CED4DA;
+$secondary-alt: #1A3B66;
+$success: #00A65A;
+$info: #555555;
+$warning: #BF7E16;
+$danger: #DD4B39;
+$white: #FFFFFF;
+$pending: $info;
+
+$theme-colors: (  
+    "primary-accent": $primary-accent,
+    "secondary-alt": $secondary-alt,
+);
+
+$body-bg: $white;
+$body-color: #333333;
+
+$font-family-sans-serif: 'Open Sans','Helvetica Neue',Helvetica,
+    Arial,sans-serif,'Apple Color Emoji','Segoe UI Emoji','Segoe UI Symbol';
+
+$h1-font-size: 1.7rem;
+$h2-font-size: 1.3rem;
+$h3-font-size: 1rem;
+$h4-font-size: 1rem;
+$h5-font-size: 1rem;
+$h6-font-size: 1rem;
+
+$small-font-size: 90%;
+$font-size-lg: 1.15rem;
+$code-font-size: 100%;
+
+$navbar-padding-y: .75rem;
+$grid-gutter-width: 20px;
+$card-spacer-y: .6rem;
+
+$list-group-item-padding-y: .6rem;
+$list-group-active-color: $body-color;
+$list-group-active-bg: $white;
+$list-group-active-border-color: rgba(#000000, .125);
+
+$dropdown-link-color: $body-color;
+$dropdown-link-hover-bg: rgba(#000000, .06);
+$dropdown-link-active-color: $dropdown-link-color;
+$dropdown-link-active-bg: rgba(#000000, .1);
+$dropdown-item-padding-x: 1rem;
+
+$navbar-brand-font-size: 35px;
+$navbar-brand-height: 35px;
+$navbar-brand-padding-y: 0;
+$navbar-dark-color: rgba($white, .7);
+$navbar-dark-hover-color: rgba($white, .9);
+$navbar-nav-link-padding-x: 0.8rem;
+
+$input-bg: #FBFBFB;
+$input-focus-bg: $white;
+$input-disabled-bg: #E0E0E0;
+$input-placeholder-color: #B4B4B4;
+
+$table-accent-bg: rgba(#000000, .02);
+$table-hover-bg: rgba(#000000, .03);
+
+$modal-backdrop-opacity: 0.3;
+$btn-font-weight: 600;
+$lead-font-weight: normal;
+
+$grid-breakpoints: (
```

Comment:
This isn't referenced anywhere and also doesn't seem like it has valid values

Reply:
This is just what was in the styles.scss originally. I haven't removed any of the previously set variables, I can remove them if you want?

Reply:
Oh ok, it's a little tricky to tell what's new and what's old. Let's just leave it alone then, we're changing enough and I'm going to presume that it was there for a reason.